drivers/clock_control/clock_stm32_ll_h7.c

```diff
@@ -298,7 +284,28 @@ static int32_t optimize_regulator_voltage_scale(uint32_t sysclk_freq)
 	defined(CONFIG_CLOCK_STM32_PLL_SRC_HSI) || \
 	defined(CONFIG_CLOCK_STM32_PLL_SRC_CSI)
```

Comment:
```suggestion
	defined(CONFIG_CLOCK_STM32_PLL_SRC_CSI) || \
	defined(CONFIG_CLOCK_STM32_PLL3_ENABLE)
```
Apply everywhere

Reply:
good catch, fixed

Reply:
@gmarull I am not seeing it fixed, did I missed something?

Reply:
I missed one entry, sorry. can you check now?